### PR TITLE
Added Minkowski metric in polar coordinates

### LINKED ISSUE #450: Add minkowski metric in polar coordinates
🐞 **Problem**
Currently minkowski metric is defined in cartesian coordinates. We need `MinkowskiPolar`.

🎯 **Goal**

The metric expression is `diag(-1, 1, r ** 2, r ** 2 * sin(theta) ** 2)`. Include `c`(speed of light) in a similar fashion to that of given below.

💡 **Possible solutions**

New addition expected in the same file as given below : 

https://github.com/einsteinpy/einsteinpy/blob/08411f159293955bc3456125aac7b98ac7c8db5f/src/einsteinpy/symbolic/predefined/minkowski.py#L7-L20

📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request and add this in comments - `Addresses #<put issue number here>`
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

### PR DESCRIPTION
**Pull request by gravitas21**
Fixes #450

### CODE REVIEW

File: src/einsteinpy/symbolic/predefined/minkowski.py [hunk 1]
```diff
@@ -1,12 +1,12 @@
-from sympy import diag, symbols
+from sympy import diag, sin, symbols
 
 from einsteinpy.symbolic import constants
 from einsteinpy.symbolic.metric import MetricTensor
 
 
 def Minkowski(c=constants.c):
```
Comment (ritzvik) [inline src/einsteinpy/symbolic/predefined/minkowski.py]: A little change
```suggestion
def MinkowskiCartesian(c=constants.c):
```

File: src/einsteinpy/symbolic/predefined/minkowski.py [hunk 2]
```diff
@@ -18,3 +18,21 @@ def Minkowski(c=constants.c):
     coords = symbols("t x y z")
     metric = diag(-1, 1 / (c ** 2), 1 / (c ** 2), 1 / (c ** 2)).tolist()
     return MetricTensor(metric, coords, "ll", name="MinkowskiMetric")
+
+
```
Comment (ritzvik) [inline src/einsteinpy/symbolic/predefined/minkowski.py]:
```suggestion
Minkowski = MinkowskiCartesian

```

Comment (ritzvik) [reply src/einsteinpy/symbolic/predefined/minkowski.py]: This is important as we don't want to break existing API

File: src/einsteinpy/symbolic/predefined/__init__.py [hunk 3]
```diff
@@ -8,5 +8,5 @@
 from .find import find
 from .godel import Godel
 from .janis_newman_winicour import JanisNewmanWinicour
-from .minkowski import Minkowski
+from .minkowski import Minkowski, MinkowskiPolar
```
Comment (ritzvik) [inline src/einsteinpy/symbolic/predefined/__init__.py]:
```suggestion
from .minkowski import Minkowski, MinkowskiCartesian, MinkowskiPolar
```

In tune with the prior suggestions

**Comment by ritzvik** [comment]
Also, after addressing the suggestions, please squash the commits

https://github.com/wprig/wprig/wiki/How-to-squash-commits

**Comment by shreyasbapat** [comment]
> Also, after addressing the suggestions, please squash the commits
> 
> https://github.com/wprig/wprig/wiki/How-to-squash-commits

Here I make your life easy: https://github.com/einsteinpy/einsteinpy/wiki/How-to-squash-commits

**Comment by gravitas21** [comment]
@ritzvik  Do let me know if I can proceed with the squash or if more changes are needed.

**Comment by shreyasbapat** [comment]
Looks good to me. Rebase and Squash into 1/2 commits.

**Comment by ritzvik** [comment]
Done n dusted!
Thanks @gravitas21

**Pull request merged**